test/lib/mayaUsd/fileio/testSaveLockedAnonLayer.py

```diff
@@ -122,11 +122,11 @@ def verifyPrims(stage):
 
     def testSaveSystemLockedAnonLayerInUSD(self):
         # Save the file. Make sure the USD edits will go to a USD file.
-        self._runTestSaveLockedAnonLayer(1)
+        self._runTestSaveSystemLockedAnonLayer(1)
```

Comment:
Was this (and one below) using the wrong function before?

Reply:
Yes! Two of the tests were not even run... thankfully, they were passing.